project-base/storefront/components/Blocks/SimpleNavigation/SimpleNavigationListItem.tsx

```diff
@@ -19,11 +19,14 @@ export const SimpleNavigationListItem: FC<SimpleNavigationListItemProps> = ({
 }) => {
     const itemImage = 'mainImage' in listedItem ? listedItem.mainImage : null;
 
+    const isWithSlash = listedItem.slug.startsWith('/');
+    const href = isWithSlash ? listedItem.slug : `/${listedItem.slug}`;
```

Comment:
I think thats what getStringWithoutLeadingSlash and getStringWithoutTrailingSlash are for 🤔

Reply:
I don't see those functions to work like they add `/` to the string.

Reply:
Yeah, that is true, but it can be used to normalize the string, such as

`const href = getStringWithoutTrailingSlash(listedItem.slug) + '/'`

Reply:
Ah, ok. Seems like cleaner implementation. Refactored. https://github.com/shopsys/shopsys/pull/2778/files#diff-2645316d358580eb697e6df274d81c7696ab210403250a32adb801f7aaea4654